<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Movie</class>
 <widget class="QDialog" name="Movie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кино</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 39, 42);
color: rgb(163, 163, 164);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 10pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>Фильм:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 12pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>Страна:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 12pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>Жанр:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 12pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>Режиссер:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 12pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>В ролях:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 12pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>Длительность:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 10pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>Жанр:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 12pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>Длительность:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 12pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>Фильм:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>1111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 9pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>В ролях:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 10pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>Страна:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 10pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>Режиссер:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>591</width>
     <height>171</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 10pt &quot;Arial&quot;;
color: rgb(163, 163, 164);
border: 1px solid #292929;
</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideRight</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 12pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>Возрастное ограничение:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 12pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>Длительность:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>521</width>
     <height>351</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 10pt &quot;Arial&quot;;
color: rgb(163, 163, 164);
border: 1px solid #292929;</string>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 26pt &quot;Arial&quot;;
color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>Схема зрительного зала</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>430</y>
     <width>131</width>
     <height>351</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 10pt &quot;Arial&quot;;
color: rgb(163, 163, 164);
border: 1px solid #292929;</string>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>180</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 12pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>Ряд</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>240</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 42);
font: 75 12pt &quot;Arial&quot;;
color: rgb(163, 163, 164);</string>
   </property>
   <property name="text">
    <string>Место</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>310</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Arial&quot;;
color: rgb(0, 170, 0);</string>
   </property>
   <property name="text">
    <string>Купить </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>270</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(163, 163, 164);</string>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(163, 163, 164);</string>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
